### PR TITLE
docs/docs-various

### PR DESCRIPTION
**Pull request by pawelfus**
Various changes, mostly related to code, but also:

- improved stacking description (missing stream and overlap)
- created examples for extending-highcharts docs - current demo for trackball doesn't work (as github blocked loading sources from rawgithub)

### CODE REVIEW

**Comment by pawelfus** [comment]
Expected visual diff: previous name in series was `Bell curve asd`, removed redundant `asd`.

File: docs/maps/create-custom-maps-for-highmaps.md [hunk 1]
```diff
@@ -49,13 +49,14 @@ Highmaps can be used for more than geographic data, and more than the featured m
...
-    
-    $(function () {
-        // Initiate the chart
-        $('#container').highcharts('Map', {
-            series: /* paste your content here */
-        });
+```js
+$(function () {
+    // Initiate the chart
+    $('#container').highcharts('Map', {
```
Comment (sebastianbochan) [inline docs/maps/create-custom-maps-for-highmaps.md]: Do we need jQuery solution?

Comment (pawelfus) [reply docs/maps/create-custom-maps-for-highmaps.md]: Probably not, note my PR does not remove jQuery from docs, it's rather focused on code formatting.

Quick search shows that there's ~10 other places where jQuery is used in docs.

Comment (TorsteinHonsi) [reply docs/maps/create-custom-maps-for-highmaps.md]: For the records, they should all be removed.

**Comment by TorsteinHonsi** [comment]
All good, except the failing CI test.

**Comment by pawelfus** [comment]
Yes, I noticed that (and `console.log()` from another demo). Will fix that today later.

File: docs/maps/create-custom-maps-for-highmaps.md (hunk 1 above)
Comment (pawelfus) [reply docs/maps/create-custom-maps-for-highmaps.md]: 👍

**Comment by pawelfus** [comment]
Note: Some jQuery references were left (e.g. in custom data parsing, or how to use Highcharts with jQuery ;) ).

Failing test (I think) was connected to CORS. Tried to change the path to be the same as in other demos (locally both work fine).

**Comment by TorsteinHonsi** [comment]
Approved, awaiting tests to pass.

**Pull request merged**